composer.lock

```diff
@@ -945,16 +945,16 @@
         },
         {
             "name": "giggsey/libphonenumber-for-php",
-            "version": "8.9.16",
+            "version": "8.10.1",
```

Comment:
Could you explain why you change composer.lock ? And if you made it in test ?

Please add more comments on commits and PR (you forget to mention this change)